rfc6962-bis.xml

```diff
@@ -1603,18 +1616,7 @@ certificates around the SCT timestamp.
       </section>
       <section title="Multiple SCTs">
```

Comment:
I don't think that's a problem, is it? I can link to one using a named
anchor.
On 1 Feb 2016 5:45 p.m., "Ben Laurie" <notifications@github.com> wrote:

> In rfc6962-bis.xml
> <https://github.com/google/certificate-transparency-rfcs/pull/119#discussion_r51452222>
> :
>
> > @@ -1603,18 +1616,7 @@ certificates around the SCT timestamp.
> >        </section>
> >        <section title="Multiple SCTs">
>
> You now have two sections with the same title..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/certificate-transparency-rfcs/pull/119/files#r51452222>
> .
>